docs/pages/extensibility/rest-api-docs.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: swagger
+title: "Extensibility: REST Api Docs Reference"
+permalink: extensibility/restapi-reference
+type: Extensibility
+abstract: 'Meshery Reference for its REST API'
+data: swagger
+---
+# REST API Resources
+
+This documentation is self-generated based on Meshery's current REST API endpoints.
+
+The API can be explored interactively using the Swagger UI Playground.
```

Comment:
A link to how to run the Swagger UI playground would be great.

Reply:
link depends on the endpoint where meshery is running, and it changes so there's no way to put it dynamically here.

Reply:
It should be sth like `mesheryURL/docs` right? So documenting it like that should work...

Reply:
Yeah, we can mention it like that.

Reply:
What is the link currently? Can this be opened up directly after running Meshery? Can any user running Meshery just go to an endpoint and access this docs?

Reply:
Yeah, it's `/docs` to open swagger-ui.

Reply:
Nope, he won't be able to access docs like that. Either 
1. We can add a permanent YAML file and have a CI update this file regularly.
or
2. User would install swagger in local and then generate the spec and then he'll log into Swagger-UI (i.e. docs)

Reply:
Have a permanent YAML file sounds good. Is there a CI issue opened up for this?

Reply:
I'll update this asap.

Reply:
Issue it opened, I'll mention both the workflow there.

Reply:
Awesome. Thank you.